linode/databases/framework_datasource_schema.go

```diff
@@ -0,0 +1,117 @@
+package databases
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/terraform-provider-linode/linode/helper/frameworkfilter"
+)
+
+var filterConfig = frameworkfilter.Config{
+	"label":   {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"region":  {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"status":  {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"type":    {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"version": {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+
+	"engine":         {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"allow_list":     {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"cluster_size":   {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeInt},
+	"created":        {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"encrypted":      {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeBool},
+	"host_primary":   {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"host_secondary": {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"id":             {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeInt},
+	"instance_uri":   {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+	"updated":        {APIFilterable: false, TypeFunc: frameworkfilter.FilterTypeString},
+}
```

Comment:
They are all marked as `APIFilterable: false`, lol? I think we can still mark these in theory API filterable fields, and wait for the fix from the API team?

Reply:
Hmm, in the interest of reducing the number of stale PRs, what if we kept these fields as non-API-filterable (which mirrors the SDKv2 implementation) and created a ticket to update them once the API changes have been released

Reply:
Yep, that will work